src/cascadia/TerminalApp/ProfileConversionTraits.cpp

```diff
@@ -0,0 +1,178 @@
+﻿// Explicit specializations for JSON conversion

```

Comment:
Can this file not be a `.h` that we include in `Profile.cpp` instead?

Reply:
It can, I just... want to figure out exactly what the division is of who owns the settings enums, where they live, and where their deserializer specializers live.

Reply:
This is the part I hate the most. I did a thing in the latest revision that might be okay.